contributing/ci-omero.txt

```diff
@@ -289,6 +275,13 @@ under the :jenkinsview:`DEV` view tab of Jenkins.
         #. Builds OMERO.server and starts it
         #. Runs the robot framework tests and collect the results
 
+    :jenkinsjob:`OMERO-DEV-merge-homebrew`
+
+        This job tests the installation of OMERO 5.3 using Homebrew
```

Comment:
maybe we can just get rid of the "5.3", none of the other 5.3.x series job descriptions specify it